src/wmtk/simplex/closed_star.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "SimplexCollection.hpp"
+
+namespace wmtk::simplex {
+SimplexCollection
+closed_star(const Mesh& mesh, const Simplex& simplex, const bool sort_and_clean = true);
```

Comment:
i'd rather just make sure that a SimplexCollection is always unique/sorted than give users this option. If you allow for users to return not-sorted-or-cleaned then some member functions of SimplexCollection do not work (in particular the union/intersection code).